laalaa/settings/base.py

```diff
@@ -98,14 +99,9 @@ def root(*x):
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.contrib.gis.db.backends.postgis",
-        "NAME": os.environ.get("DB_NAME", "laalaa"),
-        "USER": os.environ.get("DB_USERNAME", os.environ.get("DB_NAME", "postgres")),
-        "PASSWORD": os.environ.get("DB_PASSWORD", ""),
-        "HOST": os.environ.get("DB_HOST", "127.0.0.1"),
-        "PORT": os.environ.get("DB_PORT", "5432"),
-    }
+    "default": dj_database_url.config(
+        default="postgis://postgres@127.0.0.1:5432/laalaa", engine="django.contrib.gis.db.backends.postgis"
```

Comment:
Will `dj_database_url` pick up `DATABASE_URL` environment variable by default?

Reply:
Yes, the method signature is `def config(env=DEFAULT_ENV, default=None, engine=None, conn_max_age=0, ssl_require=False)`

Reply:
`DEFAULT_ENV` constant has a value of `DATABASE_URL`

Reply:
It looks like a convention for this library. I would be confused about how it gets the configuration as I like explicit declarations (they make grepping possible too).

So if it’s conventional across python projects, that’s fine, otherwise, what do you think about adding the environment variable name directly?

Reply:
>what do you think about adding the environment variable name directly

Added `env="DATABASE_URL",` to the `dj_database_url.config` call now